lib/Dialect/SV/SVOps.cpp

```diff
@@ -255,7 +255,7 @@ LogicalResult RegOp::canonicalize(RegOp op, PatternRewriter &rewriter) {
   // Check that all operations on the wire are sv.assigns. All other wire
   // operations will have been handled by other canonicalization.
   for (auto &use : op.getResult().getUses())
-    if (!isa<AssignOp>(use.getOwner()))
```

Comment:
We should have a type trait, or at least a function, which says if an op is an assignment (there are at least 6 of them in verilog)